src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -1145,6 +1145,7 @@ private void RefreshViewOnSelectionChange()
             ShowHideAllGeometryPreviewCommand.RaiseCanExecuteChanged();
             SetArgumentLacingCommand.RaiseCanExecuteChanged();
             RaisePropertyChanged("HasSelection");
+            RaisePropertyChanged("IsGeometryOperationEnabled");
```

Comment:
The `IsGeometryOperationEnabled` property is per-workspace, which means a given `WorkspaceViewModel` is only bound to the `IsGeometryOperationEnabled` property of `WorkspaceModel` that it is associated with. Since this is confined within a workspace, switching to different workspace does not affect the value of `IsGeometryOperationEnabled` because they are independent of one another. Hope that makes sense.